docs/en/03_Upgrading/01_Upgrading_project.md

```diff
@@ -1204,15 +1204,16 @@ This is a good point to commit your changes to your source control system before
 
 If you've updated your class names to use namespaces you will need to reflect those changes in any existing database fields. For example, if you've renamed your `HomePage` class to `App\HomePage` then the database `ClassName` column needs to be updated to point to the `App\HomePage` class, otherwise the CMS will tell you that the page is obsolete. This also applies to polymorphic relationships.
 
-There is no automated way to do this, but you can use the list generated in .upgrade.yml and copy it to `app/_config/legacy.yml`, removing any classes that don't extend DataObject.
+There is no automated way to do this, but you can use the list generated in `.upgrade.yml` and copy it to `app/_config/legacy.yml`, removing any classes that don't extend DataObject.
 
 ```
 SilverStripe\ORM\DatabaseAdmin:
   classname_value_remapping:
     HomePage: App\HomePage
 ```
 
-This will automatically update affected columns when you first build the database.
+Once you've mapped the class names, you need to run the UpdateLegacyClassNamesTask (`/dev/tasks/UpdateLegacyClassNamesTask`) task.
+This will update affected columns and the `ClassName` field on each DataObject table will be substituted with the namespaced classname.
```

Comment:
How about using
> ... substituted with the new classname

'new' instead of 'namespaced' as technically the new class names can be anything. Similarly in the occurrences below.

Reply:
You could also say "new FQCN" (fully qualified class name) as that's the appropriate term.